plugins/single_plugins/place.cpp

```diff
@@ -104,6 +106,15 @@ class wayfire_place_window : public wf::plugin_interface_t
         view->move(window.x, window.y);
     }
 
+    void max(wayfire_view &view, wf::geometry_t workarea)
+    {
+        wf::geometry_t window = view->get_wm_geometry();
+        window.x = workarea.x + (workarea.width);
+        window.y = workarea.y + (workarea.height);
```

Comment:
Can you explain why you're adding workarea.width/height to x/y?

Reply:
It was misguided... was not thinking it through. I've removed this in the latest commit.